### ISSUE TITLE
How to calculate the CTC Loss by jax and flax?

### ISSUE THREAD

**Issue by Du-Yao** [google/jax#5411]
I want to do the Sign Language Recognition task and Speech Recognition task by jax, and the CTC Loss is the one of the most important loss function. In Tensorflow, there are tf.nn.ctc_loos and tf.keras.backend.ctc_batch_cost APIs. 
So how can I use it in jax?

**Comment by skye** [comment]
I think this is better asked on the [flax discussions page](https://github.com/google/flax/discussions), but cc @jekbradbury in case he has anything to add.

**Comment by Du-Yao** [comment]
> I think this is better asked on the [flax discussions page](https://github.com/google/flax/discussions), but cc @jekbradbury in case he has anything to add.

Thank you very much. I have asked it on the flax discussions page as you said, thanks.
